.github/workflows/dev-build.yml

```diff
@@ -35,24 +35,19 @@ jobs:
       run: composer install
 
     - name: Build for development
-      run: PREFIX=/tmp/release make test-zip
```

Comment:
`make release` names the plugin _onOffice for WP-Websites_ which can lead to the issue, that a WP instance that has the pre-release version of the plugin installed, can update itself and overwrite it with the version from wordpress.org. This is the reason why there is the `make test-zip` target: it holds on the the plugin name _onOffice for WP-Websites *(dev)*_. `make test-zip` definitely needs to stay.

Reply:
Oh, I overlooked that release changes the name. Then we'll simply .zip it twice.